src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -976,4 +976,35 @@ object SampleWdl {
 
     override val rawInputs =  Map.empty[String, String]
   }
+
```

Comment:
Yes indeed, I'll add that test, but the preevaluateExpressionForFilenames does need updating.  I've created a separate branch where I'm working on that because it ties into some other expression evaluator refactoring